index.html

```diff
@@ -111,8 +112,65 @@ <h3 class="card-heading">Multi-Post Stories Gain + Glory</h3>
   </section>
   <!-- About section  -->
   <section id="about">
+    <img src="assets/images/ab1.png" alt="" id="about-img-1">
+    <h2 class="subheading" id="about-title">About Me</h2>
+    <p id="about-content"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+    don’t hesitate to contact me.</p>
+    <ul class="social" id="about-social">
+      <a href="https://twitter.com/aaaaamir_"><li><button class="social-btn"><i class="fa fa-2x fa-twitter"></i></button></li></a>
+      <a href="https://www.linkedin.com/in/aamir-asaram-089802213/"><li><button class="social-btn"><i class="fa fa-2x fa-linkedin"></i></button></li></a>
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-2x fa-medium"></i></button></li></a>
+      <a href="https://github.com/aamir-asaram"><li><button class="social-btn"><i class="fa fa-2x fa-github"></i></button></li></a>
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-2x fa-angellist"></i></button></li></a>
+    </ul>
 
+    <img src="assets/images/ab2.png" alt="" id="about-img-2">
+    <a href="#" id="about-btn"><button class="card-button" >Get my Resume</button></a>
```

Comment:
- [ ] [OPTIONAL] I think it was missed by the previous reviewer, so I am making it optional but please do implement these requirements to make sure your project is professional, for all the buttons on your page please implement the button interactions: Active, Hover and Pressed (see design) :point_down: 
![image](https://user-images.githubusercontent.com/97521557/235288441-fe2b23c1-0de1-43b0-ad38-b90b63a308b1.png)